app/views/conferences/_form.html.slim

```diff
@@ -0,0 +1,32 @@
+.modal.fade id='new_conference_modal' tabindex='-1' role='dialog'
+  .modal-dialog role='document'
+    .modal-content
+      = simple_nested_form_for @conference, remote: true do |f|
```

Comment:
How about ditching `@conference` and using `Conference.new` here instead?
I may be overlooking something, but I for now don't see a benefit in assigning another view variable in the controller for this modal form if it's just a `Conference.new` directly passed into the form method.

Reply:
Agreed!